pkg/provenance.go

```diff
@@ -199,6 +199,9 @@ func GenerateProvenance(name, digest, ghContext, command, envs string) ([]byte,
 						"sha1": gh.SHA,
 					},
 				},
+				{
+					URI: fmt.Sprintf("https://github.com/actions/virtual-environments/releases/tag/%s/%s", os.Getenv("ImageOS"), os.Getenv("ImageVersion")),
```

Comment:
Other git references use `git+repo.git` instead of `https://repo`. Shall we re-use the same convention here?
@asraa do you remember why we chose this format? Is it part of the specs?